packages/react/package.json

```diff
@@ -16,7 +16,7 @@
     "base.min-*"
   ],
   "scripts": {
-    "build": "rimraf lib/ && yarn run lint && yarn run tcm && rollup -c",
```

Comment:
Lint is removed?

Reply:
Yeah my bad, deleted it for debugging purposes.. Maybe @nenonja you can add it back? :)

Reply:
reverted